parsers/convert-font/README.md

```diff
@@ -0,0 +1,86 @@
+# CONVERT-FONT
+
+## Description
+
+Convert font in several formats.
+
+## Interface 
+
+```ts
+interface parser {
+  "name": "convert-font",
+  "options"?: {
+    "formats"?: Array<'woff2' | 'woff' | 'otf' | 'ttf' | 'eot'>
+    "fileNameKey"?: 'name' | 'fontFamily' | Array<string>
+    "fileNameFormat"?: 'camelCase' | 'kebabCase' | 'snakeCase';
+  }
+}
+```
+
+### Options
+
+| parameter | Require    | type      | default    | description                                       |
+| --------- | ---------- | --------- | ---------- | ------------------------------------------------- |
+| `formats`    | optional   | `Array<string>`   | `["woff2", "woff", "otf", "ttf", "eot"]` | the list of formats to convert |
+| `fileNameKey`    | optional   | `name` `fontFamily` `Array<string>`   | `name` | The key of design token used has file name. Name take the name of the design token. Font concatenate font family and font style |
```

Comment:
`fileNameKeys` instead of `fileNameKey` (the s) + change the description in `The design token's keys that will be used to create the file name. These keys will be separated by a - to create the file name.`